es-models/gdc_from_graph/case.mapping.yaml

```diff
@@ -138,7 +138,7 @@ properties:
         normalizer: clinical_normalizer
```

Comment:
(This comment is not for this line. I just need a place to leave a comment.)

Some `number` fields in the dictionary that I don't see here:
```
clinical.yaml:  days_to_death:
clinical.yaml-    description: >-
clinical.yaml-      Time interval from a person's date of death to the date of initial pathologic
clinical.yaml-      diagnosis, represented as a calculated number of days. CDE: 3165475.
clinical.yaml-    type: number
```
```
clinical.yaml:  year_of_diagnosis:
clinical.yaml-    description: >-
clinical.yaml-      Numeric value to represent the year of an individual's initial pathologic diagnosis
clinical.yaml-      of cancer. CDE: 2896960.
clinical.yaml-    type: number
```
```
clinical.yaml:  age_at_diagnosis:
clinical.yaml-    description: >-
clinical.yaml-      Age in days at which a condition or disease was first diagnosed. CDE: 3225640.
clinical.yaml-      TCGA has the age in years with CDE 2006657, this is converted by ceil(age*365.25)
clinical.yaml-      to get days.
clinical.yaml-    type: number
```
```
pathology_detail.yaml-  greatest_tumor_dimension:
pathology_detail.yaml-    $ref:
pathology_detail.yaml-      - "_terms.yaml#/greatest_tumor_dimension/common"
pathology_detail.yaml:    type: number
```

Reply:
we don't index `clinical` or `pathology_detail`. there's a ticket for adding indexing `pathology_detail`. but nothing for `clinical`. I'm not even sure we want to or have any `clinical` type data to index.